src/main/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidator.java

```diff
@@ -14,28 +14,40 @@ public boolean supports(Class<?> clazz) {
 
     @Override
     public void validate(Object target, Errors errors) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "validate" is 21 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)